### PR TITLE
Docker with nginx, remove node serve

### PR DESCRIPTION
**Pull request by cdcabrera**
* ~Multistage build for Docker with node & nginx~
* Remove express serve

updates #2

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,16 +1,18 @@
...
+
+COPY . /app
+
+RUN yarn build >/dev/null \
     && yarn cache clean
 
-USER node
-EXPOSE 3000
-CMD [ "node", "./scripts/serve" ]
+FROM nginx:alpine
```
Comment (ilyavw) [inline Dockerfile]: We're trying to use centos/rhel containers across the board, would it be possible to switch to using https://hub.docker.com/r/centos/nginx-112-centos7/ ?

Comment (ilyavw) [reply Dockerfile]: Another alternative is using s2i, maybe with https://github.com/jshmrtn/s2i-nodejs-nginx ?

Comment (cdcabrera) [reply Dockerfile]: Switching should be easy, giving it a go

Comment (cdcabrera) [reply Dockerfile]: solved! see 7ffb4b3455ea1e7ff548edf246c404c66240912c many thanks @ilyavw

**Pull request merged**